configs/zsh/zshrc.zsh

```diff
@@ -9,7 +9,7 @@ colors
 if [[ ! -f $HOME/.zinit/bin/zinit.zsh ]]; then
 	print -P "%F{33}▓▒░ %F{220}Installing DHARMA Initiative Plugin Manager (zdharma/zinit)…%f"
 	command mkdir -p "$HOME/.zinit" && command chmod g-rwX "$HOME/.zinit"
-	command git clone https://github.com/zdharma/zinit "$HOME/.zinit/bin" && \
+	command git clone https://github.com/treuille/zinit.git "$HOME/.zinit/bin" && \
```

Comment:
Why not [zdharma-continuum/zinit](https://github.com/zdharma-continuum/zinit)?

Reply:
Sorry, I didn't see a more suitable repository. Of course, it is better to use repositories accompanied by one account. Fixed the repository to the one specified above.